src/icon/supportIcon.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+import { color } from '../_utils/branding'
+import { BaseIcon, BaseIconDefaultProps, Icon } from '../_utils/icon'
+
+export const SupportIcon = (props: Icon) => (
+  <BaseIcon {...props}>
+    <g>
```

Comment:
You don't need a a group tag here since you already have the path with the whole set